explore/src/main/scala/explore/observationtree/ObsSummaryTile.scala

```diff
@@ -74,87 +78,134 @@ object ObsSummaryTable:
-    private val DurationColumnId      = ColumnId("duration")
-
-    private val PriorityColumnId      = ColumnId("priority")
-    private val RAColumnId            = ColumnId("ra")
-    private val DecColumnId           = ColumnId("dec")
-    private val TimingWindowsColumnId = ColumnId("timing_windows")
-    private val SEDColumnId           = ColumnId("sed")
-    private val ChargedTimeColumnId   = ColumnId("charged_time")
-
-    val ColumnNames: Map[ColumnId, String] = Map(
-      // Default columns
+  private val ObservationIdColumnId = ColumnId("observation_id")
+  private val GroupColumnId         = ColumnId("group")
+  private val StateColumnId         = ColumnId("state")
+  private val ScienceBandColumnId   = ColumnId("science_band")
+  private val ExpanderColumnId      = ColumnId("expander")
+  private val TargetTypeColumnId    = ColumnId("target_type")
+  private val TargetColumnId        = ColumnId("target")
+  private val ConstraintsColumnId   = ColumnId("constraints")
+  private val ConfigurationColumnId = ColumnId("configuration")
+  private val DurationColumnId      = ColumnId("duration")
+  private val RAColumnId            = ColumnId("ra")
+  private val DecColumnId           = ColumnId("dec")
+  private val SEDColumnId           = ColumnId("sed")
+  // private val ValidationCheckColumnId = ColumnId("validation_check")
+  // private val CompletionColumnId    = ColumnId("completion")
+  // private val FindingChartColumnId  = ColumnId("finding_chart")
+  // private val PriorityColumnId      = ColumnId("priority")
+  // private val TimingWindowsColumnId = ColumnId("timing_windows")
+  // private val ChargedTimeColumnId = ColumnId("charged_time")
+
+  private val ColumnNames: TreeSeqMap[ColumnId, String] =
+    TreeSeqMap(
+      ExpanderColumnId      -> " ",
       ObservationIdColumnId -> "Observation Id",
+      TargetTypeColumnId    -> "Target Type",
+      TargetColumnId        -> "Targets",
       GroupColumnId         -> "Group",
-      // ValidationCheckColumnId -> " ",
       StateColumnId         -> "State",
       ScienceBandColumnId   -> "Science Band",
-      CompletionColumnId    -> "Completion",
-      ExpanderColumnId      -> " ",
-      TargetTypeColumnId    -> " ",
-      TargetColumnId        -> "Target",
-      ConstraintsColumnId   -> "Constraints",
-      FindingChartColumnId  -> "Finding Chart",
-      ConfigurationColumnId -> "Configuration",
-      DurationColumnId      -> "Duration",
-
-      // Default hidden columns
-      PriorityColumnId      -> "Priority",
       RAColumnId            -> "RA",
       DecColumnId           -> "Dec",
-      TimingWindowsColumnId -> "Scheduling Windows",
       SEDColumnId           -> "SED",
-      ChargedTimeColumnId   -> "ChargedTime"
+      ConstraintsColumnId   -> "Constraints",
+      ConfigurationColumnId -> "Configuration",
+      DurationColumnId      -> "Duration"
+      // ValidationCheckColumnId -> " ",
```

Comment:
Commented code

Reply:
We have commented the columns that are pending to add. I just consolidated them.